src/index.html

```diff
@@ -1,14 +1,76 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="reset.css">
+    <!-- Підключення шрифта Roboto -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+
+  <body id="home">
+    <header class="header">
+      <div class="container">
+        <div class="header__inner">
+          <a class="logo" href="#home">
+            <img src="images/logo.png" alt="logo-header">
+          </a>
+          <nav class="header__nav">
+            <ul class="header__list">
+              <li class="header__item">
+                <a class="header__link" href="#">
+                Apple
+                </a>
+              </li>
+              <li class="header__item">
+                <a class="header__link" href="#">
+                Samsung
+                </a>
+              </li>
+              <li class="header__item">
+                <a class="header__link" href="#">
+                Smartphones
+                </a>
+              </li>
+              <li class="header__item">
+                <a class="header__link"
+                href="#"
+                data-qa="hover">
+                Laptops & Computers
+                </a>
```

Comment:
```suggestion
                <a 
                  class="header__link"
                  href="#"
                  data-qa="hover"
                >
                  Laptops & Computers
                </a>
```
Consider fixing indentations according to the code style guide